src/bin/travis/resources/tes.conf

```diff
@@ -7,9 +7,13 @@ Storage:
         - /cromwell-executions
         - /tmp/
 DBPath: /tmp/tes_task.db
-Schedulers:
-  Local:
-    NumWorkers: 4
+Scheduler: local
 Worker:
   LogLevel: info
-  Timeout: 1
+  Timeout: -1
+  # Funnel (TES implementation) respects resource reqs
+  # Defaults 1 cpu 2 GB ram would make centaur take too long to run
+  Resources:
+    cpus: 100
+    ram: 200
+    disk: 1000
```

Comment:
For these tests Funnel is running everything locally.  These declarations are to prevent Funnel from detecting available resources automatically so we can "trick" it into running more tests in parallel.